index.html

```diff
@@ -65,32 +66,44 @@
                 </li>
             </ul>
 
-            <ul class="table__forma">
-                <li class="forma">
-                    <img src="images/forma 1.png" alt="" class="forma__img">
-                    <p class="forma__title">
-                        Индивидуальные ценовые решения
-                    </p>
-                </li>
-                <li class="forma">
-                    <img src="images/forma 2.png" alt="" class="forma__img">
-                    <p class="forma__title">
-                        Гарантия сроков разработки
-                    </p>
-                </li>
-                <li class="forma">
-                    <img src="images/forma 3.png" alt="" class="forma__img">
-                    <p class="forma__title">
-                        Акцент на качестве результата
-                    </p>
-                </li>
-                <li class="forma">
-                    <img src="images/forma 4.png" alt="" class="forma__img">
-                    <p class="forma__title">
-                        Проекты любой сложности
-                    </p>
-                </li>
-            </ul>
+            <table class="table__forma">
+                <tr>
+                    <td class="forma">
+                        <img src="images/forma 1.png" alt="">
+                    </td>
+                    <td class="forma">
+                        <img src="images/forma 2.png" alt="">
+                    </td>
+                    <td class="forma">
+                        <img src="images/forma 3.png" alt="">
+                    </td>
+                    <td class="forma">
+                        <img src="images/forma 4.png" alt="">
+                    </td>
+                </tr>
+                <tr class="forma__subtitle">
+                    <td class="forma__title">
+                        <p class="forma__title-cell">
+                            Индивидуальные ценовые решения
+                        </p>
+                    </td>
+                    <td class="forma__title">
+                        <p class="forma__title-cell">
+                            Гарантия сроков разработки
+                        </p>
+                    </td>
+                    <td class="forma__title">
+                        <p class="forma__title-cell forma__title-cell3">
+                            Акцент на качестве результата
+                        </p>
+                    </td>
+                    <td class="forma__title">
+                        <p class="forma__title-cell">
+                            Проекты любой сложности
+                        </p>
+                    </td>
+                </tr>
+            </table>
```

Comment:
What are non-header related changes doing here and below?

Reply:
Changes back to originals